src/tv2-common/helpers/graphics/caspar/index.ts

```diff
@@ -35,65 +37,81 @@ export function GetInternalGraphicContentCaspar(
 	}
 }
 
-export function GetPilotGraphicContentCaspar(
-	config: TV2BlueprintConfig,
-	context: IShowStyleUserContext,
-	parsedCue: CueDefinitionGraphic<GraphicPilot>,
-	settings: CasparPilotGeneratorSettings,
-	engine: GraphicEngine
-): WithTimeline<GraphicsContent> {
-	const graphicFolder = config.studio.GraphicFolder ? `${config.studio.GraphicFolder}\\` : ''
-	const fileName = joinAssetToFolder(config.studio.GraphicFolder, parsedCue.graphic.name)
-	const templateData = {
-		display: 'program',
-		slots: {
-			'250_full': {
-				payload: {
-					type: 'still',
-					url: encodeURI(
-						`${config.studio.HTMLGraphics.GraphicURL}\\${fileName}${config.studio.GraphicFileExtension}`
-							.replace(/\//g, '\\') // Replace forward slash with backward slash
-							.replace(/\\/g, '\\\\') // Replace every \ with \\ and encodURI. Double backslash means the HTML template will be able to parse the string correctly. encodeURI so Caspar doesn't mangle the data.
-					)
+export class HtmlPilotGraphicGenerator extends PilotGraphicGenerator {
```

Comment:
Yes, it could be named `CasparPilotGraphicGenerator`, or the folder could be renamed to html. Just another thing that we're using more than one interchangeable terms for. (unless we ever decided to play out HTML graphics outside of Caspar, which I don't think will happen soon). This will anyway end up in another file if we make `HtmlInternalGraphicGenerator`/ `CasparInternalGraphicGenerator`.